app/src/pages/playground/UpsertPromptFromTemplate.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { Suspense, useCallback } from "react";
```

Comment:
the postfix to this file being `Dialog` would help communicate what type of component it is